### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by Logan7432** [huytd/agar.io-clone#319]
For some reason i cant eat my friends sometimes i just go through them..... merging bug hasnt been fixed after 4 months, and viruses haven't been added from 4 months, and after one thing you eat you slow down A LOT. One suggestion is please add more options into the config file. thx. and btw i will do anything for someone to code these into the game!

**Comment by IgorAntun** [comment]
Please check #12

**Comment by Logan7432** [comment]
yes, that person had the same problem but it was 4 months ago and its still not fixed :( same with everything else....

**Comment by abalabahaha** [comment]
@Logan7432 Can you please listen to what is being told to you instead of making duplicate issues?

* That issue IgorAntun linked has been resolved. Read more carefully to find out why you can't eat your friends.
* Splitting was introduced ~2 weeks ago, not 4 months ago.
* Viruses aren't on the easy side of things.
* You can change the slowBase factor in config.json to fix slowing down being too slow.
* Like what options? I'm pretty sure the config correctly corresponds to what's currently in the game.

tl;dr: The community is not your personal army. And do your research.
